app/api/files/files.ts

```diff
@@ -20,7 +21,8 @@ export const files = {
     if (index) {
       await search.indexEntities({ sharedId: savedFile.entity }, '+fullText');
     }
-    return savedFile;
+    const { __v, ...document }: WithId<FileType> & { __v?: number } = savedFile;
```

Comment:
Why is this called "document"?  Is this meant to be a "mongo document"? Is it `fileInfo`? should it just be "response"? or maked savedFile a variable and reassign it?  Not sure.

Still, if we omit the value from the model, should that not take care of the value returning in the save operation?

Reply:
Totally agree, that's a better solution, actually, there's a specific option to don't include the versionKey